ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -97,6 +99,9 @@ StatusDialog {
         StatusBaseText {
             text: qsTr("Selected to token: %1").arg(swapInputParamsForm.toTokenKey)
         }
+        StatusBaseText {
```

Comment:
we can remove it as we have the network filter on top now to see whats selected currently?

Reply:
Sure, I just thought it nicely complements that info we currently have there (esp. that it shows the chainId); up to you :)

Reply:
we have to get rid of those anyways :D

Reply:
I mean eventually.

Reply:
Yeah sure, it's temporary anyway